.github/actions/verify-submodule-pull-request/Check-Parent.ps1

```diff
@@ -3,12 +3,17 @@
     [string] $Branch
 )
 
-$url = "https://api.github.com/repos/$Repository/pulls?state=open&per_page=100"
-$titles = curl -s -H 'Accept: application/vnd.github.v3+json' $url | ConvertFrom-Json | ForEach-Object { $PSItem.title }
+$ProgressPreference = "SilentlyContinue"
+$requestParameters = @{
+    Uri = "https://api.github.com/repos/$Repository/pulls?state=open&per_page=100"
+    Method = "Get"
+    Headers = @{ Accept = "application/vnd.github.v3+json" }
+}
+$titles = (Invoke-WebRequest @requestParameters).Content | ConvertFrom-Json | ForEach-Object { $PSItem.title }
```

Comment:
Confirmed that works as expected, but the "real-world" execution is always on the default branch of the repository, right?

Reply:
This always runs for a PR, so this is fine.

No non-200 responses are expected here.

Reply:
I know, what I was asking is how errors should be handled here. For example:

`Check-Parent.ps1 -Repository "asdf/qwer"` returns
```
Invoke-WebRequest: [...]\Check-Parent.ps1:12:12
Line |
  12 |  $titles = (Invoke-WebRequest @requestParameters).Content | ConvertFro …
     |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | {"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/pulls#list-pull-requests"}
```

and an invalid branch, like `Check-Parent.ps1 -Repository "Lombiq/GitHub-Actions" -Branch "asdf"` outputs nothing, because the script exits at line 16. If those are OK/expected, then it's good to go.

Reply:
That's OK.